deployments/engine/docker-compose/dm_databases_with_tls.yaml

```diff
@@ -0,0 +1,68 @@
+version: '2.3'
+
+services:
+    dm_upstream_mysql:
+        image: mysql:5.7
+        container_name: dm_upstream_mysql
+        command:
+            - --require_secure_transport=ON
+        ports:
+            - "3306:3306"
+        volumes:
+            - ./config/mysql.cnf:/etc/mysql/conf.d/mysql.cnf
+            - /tmp/mysql1:/var/lib/mysql
```

Comment:
Is this folder only used to save certification configuration? If so, better to use a more explicit path, such as `/tmp/certs/mysql1`.

Reply:
under this folder MySQL also writes other files like binlog file, storage engine file (but we don't use them)

Reply:
Do we need to access this directory via the host machine? If not, I think we can avoid binding host path, because it may bring side effects when we run multiple tests.

Reply:
need to let DM in engine access the certificate

Reply:
maybe I should try `docker cp`